### DISCUSSION TITLE
Overwrite the email address that the contact form will send to

### DISCUSSION THREAD

**Discussion by samuel-clara** [craftcms/contact-form#264] [Q&A] [answered]
Is there a way to overwrite the email address that the contact form will send to with a hidden input on the form?
What I need is to send the mail to different emails based on a user selected value in the form.

**Comment by AugustMiller** [comment, marked as answer]
You can add hashed values to your form, and apply them via config. See [this example](https://github.com/craftcms/contact-form#dynamically-adding-email-recipients) from the docs!

In your case, you could provide a series of predefined options in a `<select>` menu:

```twig
<select name="toEmail">
  <option value="{{ 'support@domain.com'|hash }}">General Inquiry</option>
  <option value="{{ 'sales@domain.com'|hash }}">Sales</option>
  <option value="{{ 'billing@domain.com'|hash }}">Billing</option>
</select>
```
